_includes/1.0/api/json/quote.json

```diff
@@ -1,90 +1,64 @@
 {
-  "uuid": "78fcc21f-af43-4141-9edb-259e76e594aa",
   "status": "open",
-  "customer": {
-    "id": 10000,
-    "type": 1,
-    "status": 1,
-    "full_name": "OneUp",
-    "address": {
-      "street_line1": "1 Rue de Rivoli",
-      "street_line2": "",
-      "street_line3": "",
-      "postal_code": "75004",
-      "city": "Paris",
-      "country": "France",
-      "country_iso_code": "FR",
-      "locality": "",
-      "short_locality": "",
-      "full_address": "1 Rue de Rivoli 75004 Paris France"
-    },
-    "shipping_address": {
-      "street_line1": "1 Rue de Rivoli",
-      "street_line2": "",
-      "street_line3": "",
-      "postal_code": "75004",
-      "city": "Paris",
-      "country": "France",
-      "country_iso_code": "FR",
-      "locality": "",
-      "short_locality": "",
-      "full_address": "1 Rue de Rivoli 75004 Paris France"
-    },
-    "phone_number": "0612345678",
-    "email": "employee@oneup.com",
-    "website": "https://www.oneup.com/",
-    "payment_terms": {
-      "id": 1,
-      "name": "Due on receipt"
-    },
-    "currency_iso_code": "EUR",
-    "sales_tax": null,
-    "user_code": "CUS-002",
-    "opt_in_email": false,
-    "notes": "",
-    "assigned_to": {
-      "id": 10003,
-      "first_name": "Anthony",
-      "last_name": "Cadet",
-      "user_code": "EMP-001"
-    },
-    "employee_count": 1,
-    "taxpayer_id": "123456789",
-    "price_family": {
-      "id": 1,
-      "name": "All Price Families"
-    },
-    "industry": "Finance",
-    "incorporation_status": "C-Corporation",
-    "rating": "Miscellaneous",
-    "tax_id": "",
-    "is_tva_prioritized": false,
-    "accounting_category": "Local",
-    "is_account_creation_triggered": null,
-    "accounting_account": {
-      "id": 10004,
-      "number": "11100 CUS-002",
-      "name": "name",
-      "is_family": false,
-      "family_id": 6
-    },
-    "third_party_label": "",
-    "company_name": "OneUp",
-    "first_name": null,
-    "last_name": null,
-    "middle_name": null,
-    "address_optional": false,
-    "contacts": null
+  "customer_id": 10000,
+  "billing_contact_id": null,
```

Comment:
Please avoid having `null` in examples. Real values are more helpful for the developer. Look at the other examples.

Reply:
That comments apply for the whole JSON

Reply:
I tried to fill the quote as much as possible but those `null` fields still there